src/screens/SendFunds/03-Amount.js

```diff
@@ -80,6 +83,8 @@ class SendAmount extends Component<Props, State> {
       txValidationWarning: null,
       totalSpent: null,
       leaving: false,
+      useAllAmount: false,
```

Comment:
unless i miss something, this shouldn't be a state because it's already part of `transaction`

the same way we don't have `amount` in state but we use this in render():   `const amount = bridge.getTransactionAmount(mainAccount, transaction);`